RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -717,12 +965,16 @@ DAClusterizerInZT_vect::vertices(const vector<reco::TransientTrack> & tracks, co
     if(verbose_){ std::cout << "last spliting at " << 1./beta << std::endl; }
 #endif
     update(beta, tks,y, false, rho0);// make sure Tc is up-to-date
+    zorder(y);
     while(merge(y,beta)){update(beta, tks,y, false, rho0);}
     unsigned int ntry=0;
     double threshold = 1.0;
     while( split(beta, tks, y, threshold) && (ntry++<10) ){
       niter=0; 
       while((update(beta, tks,y, false, rho0)>1.e-6)  && (niter++ < maxIterations_)){}
+      zorder(y);
+      if(verbose_) dump(beta, y, tks, 2); 
```

Comment:
This is the only line of code in which "verbose_" and "VI_DEBUG" are not used contemporary